htmresearch/frameworks/nlp/model_factory.py

```diff
@@ -48,40 +82,26 @@ def getNetworkConfig(networkConfigPath):
     raise e
 
 
+
 def createModel(modelName, **kwargs):
   """
   Return a classification model of the appropriate type. The model could be any
   supported subclass of ClassficationModel based on modelName.
 
   @param modelName (str)  A string representing a supported model type:
-                            htm      : ClassificationModelHTM
-                            keywords : ClassificationModelKeywords
-                            docfp    : ClassificationModelDocumentFingerprint
-                            cioword  : ClassificationModelFingerprint
+                          HTMNetwork, htm,    : ClassificationModelHTM
```

Comment:
Yes.  You can use anything that is an `attr` of `ClassificationModelTypes`, including the original values of `createModel()` and the ones used in imbu.